### PR TITLE
More informative error message in content.create()

### LINKED ISSUE #516: content.create() should provide better error message
The error message here

https://github.com/plone/plone.api/blob/master/src/plone/api/content.py#L80-L89

should include the `content_id` and the path of the container in order to be more informative in case of migrations with spurious errors.

A better error message would be something like `plone.api.exc.InvalidParameterError: Cannot add a 'newsletter' object with id=1-15-novembre to the container /magazine/segnalazioni/unibocultura/newsletter.`

PR pending

### PR DESCRIPTION
**Pull request by zopyx**
Fixes #516

### CODE REVIEW

**Comment by zopyx** [comment]
@jenkins-plone-org please run jobs

**Review by gforcada — APPROVED** [review]

**Comment by gforcada** [comment]
I think that if you rebase the branch, the GHA errors will be gone ✨

File: news/516.feature [hunk 1]
```diff
@@ -0,0 +1 @@
+- more informative error message in plone.api.content.create() [ajung]
```
Comment (davisagli) [inline news/516.feature:1]:
```suggestion
More informative error message in plone.api.content.create() [ajung]
```

(towncrier adds the bullet point when formatting the list of changes)

**Comment by davisagli** [comment]
Rebased; running tests again.

@jenkins-plone-org please run jobs

**Pull request merged**